components/features/Dashboard/Applications/VideoConference/VideoConference.tsx

```diff
@@ -24,47 +29,36 @@ const Spinner = () => {
   );
 };
 
-const Conference: React.FC = () => {
-  const [roomJoinable, setRoomJoinable] = useState(true);
-  const connState = useRef({ joining: false, leaving: false });
+const Conference: React.FC<ConferenceProps> = ({ data }) => {
+  const [loading, setLoading] = useState(false);
   const conference = useHMSActions();
   const isPreviewReady = useHMSStore(selectIsInPreview);
   const isInCall = useHMSStore(selectIsConnectedToRoom);
-  const { data, error, isValidating, mutate } = useSWRImmutable(
-    "wizzy",
-    getRoomToken,
-    { errorRetryCount: 0 }
-  );
 
-  const joinCall = async () => {
-    connState.current.joining = true;
-    await conference.join({ userName: "wizzy", authToken: data?.token! });
-    connState.current.joining = false;
+  const enterConference = async () => {
+    setLoading(true);
+    await conference
+      .join({ userName: "Rashad Aziz", authToken: data?.token })
```

Comment:
Hello, Rashad!
Just to make sure, is "Rashad Aziz" this just a place holder?
If yes, I believe it is better to store it in a separate variable such that it will be more clear.

Reply:
Hi jo, yes, of course! This is just a temporary variable until we integrate our authentication, in which we will grab the userName from there.